dom/events/passive-default/resources/default-passive-scrolling.js

```diff
@@ -0,0 +1,40 @@
+function waitFor(condition) {
+  const MAX_FRAME = 200;
+  return new Promise((resolve, reject) => {
+    function tick(frames) {
+      // We requestAnimationFrame either for 200 frames or until condition is
+      // met.
+      if (frames >= MAX_FRAME)
+        reject("Condition did not become true after 200 frames");
```

Comment:
```suggestion
        reject(new Error("Condition did not become true after 200 frames"));
```

This is best practice in general and will ensure you get stack traces on failures.